#!/usr/bin/env python

PACKAGE = 'kubot_duo_box'

from dynamic_reconfigure.parameter_generator_catkin import ParameterGenerator, str_t, double_t, bool_t, int_t

gen = ParameterGenerator()

gen.add("led_pixel", int_t, 0, "led pixel num", 11, 0, 255)
gen.add("servo_max", int_t, 0, "servo motor max angule", 179, 0, 180)
gen.add("servo_min", int_t, 0, "servo motor max angule", 5, 0, 180)

exit(gen.generate(PACKAGE, "kubot_duo_box", "kubot_driver"))

#KUBOT_PARAMETER_CFG_